src/azul/service/avro_pfb.py

```diff
@@ -329,6 +329,59 @@ def pfb_schema_from_replicas(replicas: Iterable[JSON]
     return keys, _avro_pfb_schema(values)
 
 
+def pfb_schema_from_anvil_schema(anvil_schema: JSON
+                                 ) -> tuple[Sequence[str], JSON]:
+    """
+    Create a PFB schema for replicas of entities described by the AnVIL schema.
+    :param anvil_schema: A JSON document describing the tables and columns of
+                         the AnVIL metadata as stored in underlying repository.
+    :return: A tuple of
+             1. The set of entity types defined by the PFB schema
+             2. The PFB schema itself
```

Comment:
```suggestion
    Create a PFB schema for replicas of entities described by the AnVIL schema.
    
    :param anvil_schema: A JSON document describing the tables and columns of
                         the AnVIL metadata as stored in underlying repository.
                         
    :return: A tuple of
             1. The set of entity types defined by the PFB schema
             2. The PFB schema itself
```

Reply:
Function was inlined and its docstring removed.